app/models/document.rb

```diff
@@ -121,7 +121,11 @@ def change_history
   end
 
   def update_type
-    @update_type || "major"
+    if self.draft? && @update_type != "republish"
```

Comment:
think the second half should no longer be here: even if a draft is republished it still needs to be a major update so that email alerts are sent